ui/src/components/WebRTCVideo.tsx

```diff
@@ -344,153 +335,45 @@ export default function WebRTCVideo() {
     sendAbsMouseMovement(0, 0, 0);
   }, [sendAbsMouseMovement]);
 
-  // Keyboard-related
```

Comment:
This code may have to be restored and guarded to be used if the browser is talking to an old device... can that happen with the online (hosted) UI?

Reply:
I moved all this handling into _useKeyboard.ts_